main.go

```diff
@@ -180,19 +234,13 @@ func (s *service) startTechnicalEndpoint() {
 		config.ValidateConfigurationHTTPHandler,
 	)
 	go func() {
-		srv := &graceful.Server{
-			Server: &http.Server{
-				Addr:           port,
-				Handler:        serveMuxHandler,
-				MaxHeaderBytes: 512,
-				WriteTimeout:   TechnicalEndpointGeneralTimeout,
-				ReadTimeout:    TechnicalEndpointGeneralTimeout,
-			},
-			Timeout:      TechnicalEndpointGeneralTimeout,
-			TCPKeepAlive: 1 * time.Minute,
-			Logger:       graceful.DefaultLogger(),
+		srv := &http.Server{
+			Addr:           port,
+			Handler:        serveMuxHandler,
```

Comment:
`port` variable is related to shared config. It will be nice to refresh `http.Server` in  `startTechnicalEndpoint`